docs/standaard/catalogi/index.md

```diff
@@ -317,6 +317,55 @@ De voorwaarden waaronder een correctie uitgevoerd mag worden zijn:
+| catalogi   | 1.3.0        | catalogussen                  | besluittypen           | Indirect | catalogi   | besluittypen                  |                                         |
+| catalogi   | 1.3.0        | catalogussen                  | informatieobjecttypen  | Indirect | catalogi   | informatieobjecttypen         |                                         |
+| catalogi   | 1.3.0        | eigenschappen                 | catalogus              | Indirect | catalogi   | catalogussen                  |                                         |
+| catalogi   | 1.3.0        | eigenschappen                 | zaaktype               | Indirect | catalogi   | zaaktypen                     |                                         |
+| catalogi   | 1.3.0        | eigenschappen                 | statustype             | Indirect | catalogi   | statustypen                   |                                         |
+| catalogi   | 1.3.0        | informatieobjecttypen         | catalogus              | Indirect | catalogi   | catalogussen                  |                                         |
+| catalogi   | 1.3.0        | informatieobjecttypen         | zaaktypen              | Indirect | catalogi   | zaaktypen                     |                                         |
+| catalogi   | 1.3.0        | informatieobjecttypen         | besluittypen           | Indirect | catalogi   | besluittypen                  |                                         |
+| catalogi   | 1.3.0        | resultaattypen                | zaaktype               | Indirect | catalogi   | zaaktypen                     |                                         |
+| catalogi   | 1.3.0        | resultaattypen                | catalogus              | Indirect | catalogi   | catalogussen                  |                                         |
+| catalogi   | 1.3.0        | resultaattypen                | besluittypen           | Indirect | catalogi   | besluittypen                  |                                         |
+| catalogi   | 1.3.0        | resultaattypen                | informatieobjecttypen  | Indirect | catalogi   | informatieobjecttypen         |                                         |
+| catalogi   | 1.3.0        | statustypen                   | zaaktype               | Indirect | catalogi   | zaaktypen                     |                                         |
+| catalogi   | 1.3.0        | statustypen                   | catalogus              | Indirect | catalogi   | catalogussen                  |                                         |
+| catalogi   | 1.3.0        | statustypen                   | eigenschappen          | Indirect | catalogi   | eigenschappen                 |                                         |
+| catalogi   | 1.3.0        | zaakobjecttypen               | objecttype             | Indirect | catalogi   | objecttypen                   |                                         |
```

Comment:
De rij met het veld "objecttype" kan niet worden ge-expandeerd en moet worden weggelaten. Immers er bestaat geen resource "objecttypen" in de catalogi API.